packages/big-design/src/components/Table/types.ts

```diff
@@ -9,15 +9,25 @@ export interface TableSelectable<T> {
   onSelectionChange(selectedItems: T[]): void;
 }
 
+export type TableSortDirection = 'ASC' | 'DESC';
```

Comment:
I've seen some tables out there that have 3 states: Default, Ascending, and Descending.

As an example of what I'm talking about, see the `Birth Place` column on this table: https://material-ui.com/components/tables/#material-table.

Is our sortable columns only going to have 2 states? Is this what design intended?

Reply:
You can achieve that third state by not setting a `sortable.columnHash`.

Reply:
When you remove that key/val, the arrow drops off.

Reply:
Yeah since it's no longer `ASC` or `DESC`, it is kind of confusing pressing it and the arrow not changing. The user doesn't really know / cares if its the default or not 🤔

Reply:
As a user, I would care that the arrow appears, specifically to show which order the column is sorted in. Most tables I've seen or used has 3 different states out of the box. Since you mentioned that design is still in-between these patterns/designs, is this something we want to wait for design to finish? If not, we can accept only two states for now.

Reply:
Can you link me to the tables you are referring to other than material-ui?

I don't agree with a third state, however, adding it later is an additive an easy change.

Reply:
Carbon is probably the best one I found:
https://www.carbondesignsystem.com/components/data-table/code

Both Atlaskit and Polaris intially default to no sorting, then it toggles sort on click.
https://atlaskit.atlassian.com/packages/core/dynamic-table
https://polaris.shopify.com/components/lists-and-tables/data-table#navigation

I also noticed with that with those two that it removes the arrow when sorting another column. I tested your PR locally and noticed that it sorts the column, but the arrow stays on the initial column.

Reply:
>I also noticed with that with those two that it removes the arrow when sorting another column. I tested your PR locally and noticed that it sorts the column, but the arrow stays on the initial column.

I think this is due to not being able to pass in multiple columnHashes and sort functions.

Reply:
So we have the same approach as Atlaskit and Polaris then, right?

To move the icon to the new column you need to change `columnHash` to the column that you need.

Reply:
> So we have the same approach as Atlaskit and Polaris then, right?

Atlaskit and Polaris default it to unsorted, as in unsorted data returned from an API. Whereas your implementation defaults the sort to `ASC` or `DSC` based off the `direction`, there's no default unsorted state.

> To move the icon to the new column you need to change columnHash to the column that you need.

Gotcha, then that's an implementation detail.

Reply:
That just depends on how you use it, you can do an unsorted state by just not passing a `columnHash` since you aren't sorting any column.

Reply:
So if we are starting to restrict functionality, why are we not baking that into the sort functionality itself rather than letting the app have to think about the logic of switching the `columnHash` and the sort function?

Reply:
Because it's a 1-liner for the app and if we add it to the library then it will have to have its own state and can potentially be toggling between ASC / DESC without the app actually doing the sorting and getting out of sync.

Reply:
🤷‍♀ I'll take it